kernel/Base/PhysicalElement.h

```diff
@@ -240,10 +246,10 @@ class PhysicalElement final {
     LinearAlgebra::MiddleSizeVector solutionDiv;
 
     Geometry::PointPhysical<DIM> pointPhysical;
-    Geometry::Jacobian<DIM, DIM> jacobian, transposeJacobian,
-        inverseTransposeJacobian;
+    mutable Geometry::Jacobian<DIM, DIM> jacobian, transposeJacobian;
+    Geometry::Jacobian<DIM, DIM> inverseTransposeJacobian;
     double jacobianAbsDet;
-    double jacobianDet;
+    mutable double jacobianDet;
```

Comment:
I know you have a warning earlier, but that can get problematic if multiple threads use it.

Reply:
Is that because people would expect const-members to be completely thread-safe?

Besides, PhysicalElement is already a class that I would not think makes much sense in sharing between threads.

Reply:
Yeah, then please go ahead.